examples/tests/invalid/class.test.w

```diff
@@ -5,6 +5,17 @@ class C1 {
 // ^ Preflight field "x" is not initialized
 }
 
+class C1_1 {
+  x:num;
+  new() {
+    () => {
+      // This doesn't count as an initialization because it's in a closure that might not be called
+      // and should produce an error.
+      this.x = 1;
```

Comment:
Since this will be a somewhat subtle breaking change, could you compile the projects in https://github.com/winglang/winglibs to preemptively prepare any needed fixes? Just to avoid pagerduty alerts :)

Reply:
done